cross/build-rootfs.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+
+usage()
+{
+    echo "Usage: $0 [BuildArch]"
+    echo "BuildArch can be: arm, arm64"
+
+    exit 1
+}
+
+__CrossDir=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+__InitialDir=$PWD
+__BuildArch=arm
+__UbuntuArch=armhf
+__UbuntuRepo="http://ports.ubuntu.com/"
+__UbuntuPackages="build-essential libunwind8-dev gettext symlinks liblttng-ust-dev libicu-dev zlib1g-dev libssl-dev libcurl4-openssl-dev libz-dev libcrypto++-dev"
```

Comment:
@weshaggard Does it sound reasonable for the script needed to create cross-build environment to live in buildtools repo?